changelogs/unreleased/3111-ashish-amarnath

```diff
@@ -0,0 +1 @@
+[WIP] ✨ Add more E2E tests and improvement
```

Comment:
1) Let's not include WIP in changelogs
2) I don't think we need a changelog for tests :)

Reply:
Yes, the "WIP" in the changelog was accidental. Yeah, the changelog is not required. Will remove.